contribs/dvrp/src/test/java/org/matsim/contrib/dvrp/examples/onetaxi/RunOneTaxiWithPrebookingExampleIT.java

```diff
@@ -239,7 +239,7 @@ private static void assertRequestScheduledEvent(Map<Id<Person>, PassengerRequest
 		PassengerRequestScheduledEvent event = events.get(Id.createPersonId(personId));
 		assertThat(event.getVehicleId().toString()).isEqualTo("taxi_one");
 		assertThat(event.getPickupTime()).isCloseTo(pickupTime, Offset.offset(0.01));
-		assertThat(event.getPersonId().toString()).isEqualTo(personId);
+		assertThat(event.getPersonIds().stream().findAny().orElseThrow().toString()).isEqualTo(personId);
```

Comment:
I would rather assert that size is 1 and then take the first element (BTW. after changing to a list, it will be easier to take the first element).